doc/nrf/releases/release-notes-latest.rst

```diff
@@ -141,6 +141,52 @@ Matter (Project CHIP)
 
   * Renamed occurrences of Project CHIP to Matter.
 
+Bluetooth Mesh
+--------------
+
+* Added:
+
+  * Support for vendor specific mesh model :ref:`bt_mesh_silvair_enocean_srv_readme`.
+  * API function to manually store pending RPL entries in the persistent storage without waiting for the timeout.
+
+* Updated:
+
+  * Fixed an issue by :ref:`bt_mesh_light_xyl_srv_readme` publishes its state values that was loaded from flash after power up.
+  * Replaced the linked list of scene entries in the model contexts, with a lookup in ROM-allocated scene entries for each model in the composition data when storing and restoring.
+  * Updated so the transition pointer can be NULL, if no transition time parameters are provided in APIs.
+  * Renamed Kconfig option ``CONFIG_BT_MESH_LIGHT_CTRL_STORE_TIMEOUT`` to :option:`CONFIG_BT_MESH_MODEL_SRV_STORE_TIMEOUT`, and default value is set to 0.
+  * Fixed an issue by added a timer that allows the :ref:`bt_mesh_light_ctrl_srv_readme` to resume operation after a certain delay.
```

Comment:
Again, are we describing the solution to an issue that is not mentioned? And should we mention it, or are we ok with this?

Reply:
I think it sometime is better to describe what the solution is. I change this one any way.